Makefile

```diff
@@ -32,7 +32,7 @@ COMMON_ARGS += --frontend-opt build-arg:TOOLCHAIN_IMAGE=$(TOOLCHAIN_IMAGE)
 COMMON_ARGS += --frontend-opt build-arg:SHA=$(SHA)
 COMMON_ARGS += --frontend-opt build-arg:TAG=$(TAG)
 
-all: ci lint test kernel initramfs rootfs osctl-linux-amd64 osctl-darwin-amd64 osinstall-linux-amd64 installer
```

Comment:
unsure if we should leave these around for local calls

Reply:
yeah, maybe at least the `ci`, but otherwise we won't be able to breakout `lint` and `test` in parallel builds.